CONTRIBUTING.md

```diff
@@ -151,6 +151,7 @@ Common checks that may occur in our repositories:
 
 1. Travis CI - where our automated tests are running
 2. Approval Required - Github enforces at least one person approve a pull request. Also, all reviewers that have chimed in must approve.
+3. Required Labels - an appropriate [semver](https://semver.org/) label (see [release.yml](https://github.com/samvera/hyku/blob/main/.github/release.yml))
```

Comment:
delete if we end up deleting release.yml

Reply:
keeping to keep in line with hyku